### PR TITLE
SO-112: To Install helmcharts through Samples operator

### PR DESCRIPTION
**Pull request by fbm3307**
This PR is to update the Cluster Samples Operator CRD to have the Helmchart List . This is Part of Samples operator Downsizing Phase-2, where the S2i Sample helmcharts can be installed via CSO.
Refernce -[SO-112](https://issues.redhat.com/browse/SO-112)

### CODE REVIEW

File: samples/v1/types_config.go [hunk 1]
```diff
@@ -64,6 +64,17 @@ type ConfigSpec struct {
...
+	// helmchart list specifies the list of helmcharts that the samples operator will install
+	HelmChartList []HelmChart `json:"helmChartList,omitempty" protobuf:"bytes,7,opt,name=helmChartList"`
+}
+
+type HelmChart struct {
+	URL string `json:"url,omitempty"`
+
+	Namespace string `json:"namespace,omitempty"`
+
+	ReleaseName string `json:"releaseName,omitempty"`
```
Comment (bparees) [inline samples/v1/types_config.go:77]: godoc these fields please.  Also is there any validation we can put in place?

should these be required fields?

**Comment by fbm3307** [comment]
closing this in favor of https://github.com/openshift/api/pull/1636
 as this PR is very old and it is causing problems in testcases

**Pull request closed without merging**